components.yaml

```diff
@@ -37,10 +37,10 @@ components:
     metadata: v4.0.2
   multus-dynamic-networks:
     url: https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller
-    commit: f40f85bc20f361aeeac7b4b04aa6d289c8e130f5
+    commit: 21b6dee0542e5f402a19b86b2b925f3bd7940bdb
     branch: main
     update-policy: tagged
```

Comment:
@maiqueb @phoracek 
on release 0.89 / release 0.85 of CNAO we have networks-controller with update-policy static
so we need to backport manually all the PRs like this one

maybe we should consider please to create branches on multus-dynamic-networks-controller ?
unless it is fine to change to tagged (but then it will take every tag, because the repo doesnt use branches)

Reply:
Update it to tagged if needed.

Reply:
https://github.com/kubevirt/cluster-network-addons-operator/pull/1728

will create same for 0.85 once it is accepted